app/views/simple_calendar/_week_calendar.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="simple-calendar">
+  <div class="calendar-heading">
+    <%= link_to t("simple_calendar.previous", default: "Previous"), calendar.url_for_previous_view %>
+    <% if calendar.number_of_weeks == 1 %>
+      <span class="calendar-title"><%= t("simple_calendar.week", default: "Week") %> <%= calendar.week_number %></span>
+    <% else %>
+      <span class="calendar-title"><%= t("simple_calendar.week", default: "Week") %> <%= calendar.week_number %> - <%= calendar.end_week %></span>
+    <% end %>
+    <%= link_to t("simple_calendar.next", default: "Next"), calendar.url_for_next_view %>
+  </div>
+
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <% date_range.slice(0, 7).each do |day| %>
+          <th><%= t("date.abbr_day_names")[day.wday] %></th>
+        <% end %>
+      </tr>
+    </thead>
+
+    <tbody>
+      <% date_range.each_slice(7) do |week| %>
+        <tr>
+          <% week.each do |day| %>
+            <%= content_tag :td, class: calendar.td_classes_for(day) do %>
+              <% if defined?(Haml) && respond_to?(:block_is_haml?) && block_is_haml?(passed_block) %>
```

Comment:
Haml!  /me shakes fist at haml!

I'm guessing these `simple_calendar` views are generated?

Reply:
HAML is just an alternative some people use to html.erb views, in their rails projects.

Since it's generated from the gem, i'd say it's fine.  Boilerplate gonna boiler plate.